scipy/sparse/_construct.py

```diff
@@ -481,9 +481,9 @@ def kronsum(A, B, format=None):
     return (L+R).asformat(format)  # since L + R is not always same format
 
 
-def _compressed_sparse_stack(blocks, axis):
+def _compressed_sparse_stack(blocks, axis, return_spmatrix):
```

Comment:
This change causes a test failure when `_compressed_sparse_stack` is called here

https://github.com/scipy/scipy/blob/d55e2f905fc1b7f4a1f6a5f841ca7fb0ba505ae6/scipy/sparse/tests/test_construct.py#L542

without the required third argument.

Reply:
Ahh... very strange. So we have tests that aren't running regularly?

Anyway -- I should have caught that. :/ new (small) PR coming.

Reply:
Thanks for catching it!  And for pointing out exactly where and what the failure is. :)
I learned something today! :)